validation/telemetry/deletion-request.4.payload.fail.json

```diff
@@ -0,0 +1,26 @@
+{
+  "type": "deletion-request",
+  "id": "d7754449-8cb1-4df6-b07b-c60dab6f5d66",
+  "creationDate": "2018-07-09T14:13:14.497Z",
+  "version": 4,
+  "application": {
+    "architecture": "x86-64",
+    "buildId": "20180709105613",
+    "name": "Firefox",
+    "version": "63.0a1",
+    "displayVersion": "63.0a1",
+    "vendor": "Mozilla",
+    "platformVersion": "63.0a1",
+    "xpcomAbi": "x86_64-gcc3",
+    "channel": "default"
+  },
+  "payload": { 
```

Comment:
This test should include a clientId to make sure it's the payload part that's causing a failure.